client/Components/ui/dropdown-menu.tsx

```diff
@@ -55,7 +55,8 @@ const DropdownMenuContent = React.forwardRef<
   React.ElementRef< typeof DropdownMenuPrimitive.Content >,
   React.ComponentPropsWithoutRef< typeof DropdownMenuPrimitive.Content >
 >( ( { className, sideOffset = 4, ...props }, ref ) => (
-  <DropdownMenuPrimitive.Portal container={ window.agentwp }>
+  <DropdownMenuPrimitive.Portal
+    container={ window.agentwp.querySelector( '#agentwp-chat-shadow-root' )?.shadowRoot }>
```

Comment:
can we use a more dynamic query here? if the container changes or there are multiple apps on the same page this won't work

Reply:
`#agentwp-chat-shadow-root` is the actual shadow which is inside `window.agentwp`, which is also targeted using the element ID `document.getElementById( 'agentwp-admin-chat' );`. 

There will always be only one, no?
----
I can also make it more dynamic, but I do not know what you mean by that :)
Should I use a class instead of an ID, or?

Reply:
Yes there can be more than one app on the page (Dashboard Widget + Chat for example). So the definition of the container should happen when the react app is initialized and just passed down. I'll make a quick commit for this.